From ae90969fd5e47ab47d9bbbef4fb84ba6eb65cb72 Mon Sep 17 00:00:00 2001
From: Li Wang <li.wang@windriver.com>
Date: Tue, 25 Aug 2015 15:40:20 +0800
Subject: [PATCH 37/68] arm: axxia: Fix compiler warning from PCI driver

the patch comes from:
git://git.yoctoproject.org/linux-yocto-3.10
commit 64ec062183df8169288fad1b62f1373080f0e1b8 upstream

Fix missing return value in PCI drivers probe function.

Signed-off-by: Anders Berg <anders.berg@intel.com>
Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
Signed-off-by: Li Wang <li.wang@windriver.com>
---
 arch/arm/mach-axxia/pci.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-axxia/pci.c b/arch/arm/mach-axxia/pci.c
index f85b047..65e2f39 100644
--- a/arch/arm/mach-axxia/pci.c
+++ b/arch/arm/mach-axxia/pci.c
@@ -701,7 +701,7 @@ static int axxia_pcie_setup(int portno, struct pci_sys_data *sys)
 		writel(INT1_DOORBELL,
 			port->regs + PCIE_INT1_ENABLE);
 		release_resource(&sys->io_res);
-		return;
+		return 0;
 	}
 
 	/* Make sure the link is up */
-- 
1.7.5.4

